networking/hardware_networks/uninstalling-sriov-operator.adoc

```diff
@@ -0,0 +1,57 @@
+[id="uninstalling-sriov-operator"]
+= Uninstalling the SR-IOV Network Operator
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sr-iov-operator
+
+toc::[]
+
+To uninstall the SR-IOV Network Operator, you must first delete any running SR-IOV workloads. You must also delete the webhooks after uninstalling the operator.
+
+.Prerequisites
```

Comment:
You need to convert this content into a module and include it in the assembly. (Make sure you apply the `:_content-type: PROCEDURE` tag according to the [contrib guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#content-type-attributes).